docs/source/conf.py

```diff
@@ -101,6 +101,8 @@
 html_theme_options = {
     "github_url": "https://github.com/alteryx/evalml",
     "twitter_url": "https://twitter.com/AlteryxOSS",
+    "collapse_navigation": True,
```

Comment:
This turns out to be pretty important. 

https://sphinx-rtd-theme.readthedocs.io/en/stable/configuring.html#confval-collapse_navigation

Reply:
🤯  amazing.

I wonder if the navigation depth also helped